src/typescript-service.ts

```diff
@@ -1544,8 +1557,9 @@ export class TypeScriptService {
 		}
 		configuration.getHost().addFile(fileName);
 		configuration.syncProgram();
-
-		return configuration.getProgram().getSourceFile(fileName);
+		// update program object
+		program = configuration.getProgram();
+		return program ? program.getSourceFile(fileName) : null;
```

Comment:
... that would simplify this to `return program && program.getSourceFile(fileName)`